qiskit/extensions/standard/cswap.py

```diff
@@ -55,12 +56,23 @@ def inverse(self):
         """Invert this gate."""
         return FredkinGate()  # self-inverse
 
+    def to_matrix(self):
+        """Return a Numpy.array for the Fredkin (CSWAP) gate."""
+        return numpy.array([[1, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
Might need to be aligned to qiskit's qubit indexing convention (I think this is causing the test failures):
```
>>> qk.quantum_info.Operator(FredkinGate()).data.real
array([[1., 0., 0., 0., 0., 0., 0., 0.],
       [0., 1., 0., 0., 0., 0., 0., 0.],
       [0., 0., 1., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 1., 0., 0.],
       [0., 0., 0., 0., 1., 0., 0., 0.],
       [0., 0., 0., 1., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 1., 0.],
       [0., 0., 0., 0., 0., 0., 0., 1.]])
```

Reply:
done